FogCreek/Glitch-Community
# Quixotic rocket: SSR transpile on startup, indicate initial vs retranspile

## Links
https://quixotic-rocket.glitch.me/
https://github.com/FogCreek/Glitch-Community-Work/issues/393

## Changes:
- Run a transpile right after server startup (in a setImmediate so webpack has a chance to start)
- Print 'Retranspiling' if this isn't the first transpile after server start
- Slight readme change to include a full file path

## How To Test:
1. Join the  project and run `refresh`, the project will transpile without loading a page
2. Edit something in the src/ directory (you don't have to wait for webpack)
3. Try loading the homepage, it should print 'retranspiling'
